from libp2p.io.exceptions import (
    IOException,
)


class RawConnError(IOException):
    pass
